install.zasilkovna.php

```diff
@@ -540,18 +551,18 @@ private function migratePricingRules() {
         $this->migratingPricingRules = false;
     }
 
-	/**
-	 * Called on uninstallation
-	 *
-	 * @param   JAdapterInstance|PluginAdapter $adapter The object responsible for running this script
-	 */
-	public function uninstall($adapter) {
-		$db = JFactory::getDBO();
-		$q = "DELETE FROM #__virtuemart_adminmenuentries WHERE `name` = 'zasilkovna';";
-		$db->setQuery($q);
-		$db->execute();
+    /**
+     * Called on uninstallation
+     *
+     * @param   JAdapterInstance|PluginAdapter $adapter The object responsible for running this script
+     */
+    public function uninstall($adapter) {
+        $db = JFactory::getDBO();
+        $q = sprintf("DELETE FROM #__virtuemart_adminmenuentries WHERE `name` = '%s';", self::VM_MENU_ENTRY);
```

Comment:
Odstranění položky "Packeta" z menu bych dal do jedné metody, aby odstranění této položky ze všech menu kde jsou bylo pohromadě v jedné metodě a dělo se to na jednom místě.